tools/debian/rules

```diff
@@ -5,8 +5,10 @@ DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 # only ship deprecated PatternFly API for stable releases (backports); Debian sid/testing has no VERSION_ID
 SHIP_PF_API = $(filter $(shell . /etc/os-release; echo $${VERSION_ID:-unstable}),10 20.04)
 
-# this is an emulated architecture for now, and too slow to run expensive unit tests
-ifeq ($(shell dpkg-architecture -qDEB_BUILD_ARCH),riscv64)
+# riscv is an emulated architecture for now, and too slow to run expensive unit tests
+# hppa's threading is absurdly slow (#981127)
+SLOW_ARCHES = $(filter $(shell dpkg-architecture -qDEB_BUILD_ARCH),riscv64 hppa)
```

Comment:
*squint*

Fine with this change in theory, but I wonder if you could find a more readable way to express it...

I guess we don't have any kind of `in` verb here unfortunately :/

Reply:
Heh, yeah.. I modelled it after the other `ifneq()` bits we have (and previously had) in rules. Makefile syntax is so arcane that I usually resort to copy&paste of proven snippets than trying to invent something new :-/